dips/dip-20.md

```diff
@@ -635,12 +569,12 @@ state of the blockchain that should be kept. These are:
 # <a name="operational_things">Operational Restrictions</a>
 ---
 
-When deciding the on-chain currency code for a currency,
-the module, as well as the type name chosen for the currency must match the on-chain currency
-code chosen for the currency. e.g., if you want to use currency code `"ABC"` for a
-currency, then the fully qualified Move type that is used for that currency on-chain
-_must_ be `0x1::ABC::ABC`.
-
-Because of this, and in addition:
-* An on-chain currency code must be a valid Move identifier; and
-* It can consist solely of uppercase alphanumeric characters.
+When deciding on the on-chain currency code for a currency, the module as
```

Comment:
Several places here refer to the "on-chain currency code". There's no distinction between on-chain and off-chain currency codes, is there? Each currency has a single currency code that must be used consistently throughout the system. I suggest dropping "on-chain" for at least most of these references.